### DISCUSSION TITLE
Where's the util.js file?

### DISCUSSION THREAD

**Discussion by donaldG** [twbs/bootstrap#35698] [Q&A]
New to using Bootstrap, trying to get a carousel up and see that the main Bootstrap docs say you need util.js imported. I don't see that file in this repo (../js/dist/???). I'm *assuming* it's been bundled into something somewhere and I'm just new so I don't know it, ha. But maybe it's accidentally missing?

**Comment by CFarhad** [comment]
why do you need `util.js`
if you want use bootstrap ems module then read this [document](https://getbootstrap.com/docs/5.0/getting-started/javascript/#using-bootstrap-as-a-module) from bootstrap website.

please describe more
